### PR TITLE
Improved Unit-Test Coverage for Issue #703

### PR DESCRIPTION
**Pull request by aarongalang**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
Adds unit-tests to pkg folder for improved coverage

**Which issue(s) this PR fixes**:
#703 

Fixes #

**Requirements**:
  
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:
This PR had to be created because my previous pr had a commit that used my local computer as the author, rather than my git user id. As a result, only that one commit did not have my cla, causing the pr to be mergeable
please see pr #723 for actual commit messages
**Release note**:
```
none
```

### CODE REVIEW

**Comment by aarongalang** [comment]
accidentally changed the non-test files, undoing them now

File: pkg/blob/blob.go [hunk 1]
```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 The Kubernetes Authors.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
```
Comment (mukhoakash) [inline pkg/blob/blob.go:7]: revert the changes in this file

Comment (mukhoakash) [reply pkg/blob/blob.go:7]: Are these created by lint?

File: pkg/blob/controllerserver.go [hunk 2]
```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2017 The Kubernetes Authors.
-
```
Comment (mukhoakash) [inline pkg/blob/controllerserver.go:3]: Revert, unless created by lint fixes.

File: pkg/blob/nodeserver.go [hunk 3]
```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2017 The Kubernetes Authors.
-
```
Comment (mukhoakash) [inline pkg/blob/nodeserver.go:3]: Same. Revert unless created by lint fix.

File: test/utils/credentials/credentials.go [hunk 4]
```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 The Kubernetes Authors.
-
```
Comment (mukhoakash) [inline test/utils/credentials/credentials.go:3]: Same. Revert unless created by lint fix.

File: pkg/blob/blob.go (hunk 1 above)
Comment (aarongalang) [reply pkg/blob/blob.go:7]: yeah i'm not sure why those ended up there... will revert momentarily

Comment (aarongalang) [reply pkg/blob/blob.go:7]: double checked my code, those dashes aren't there, might have been created by lint fixes as you said

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve
